docs/pages/product/configuration/data-sources/aws-redshift.mdx

```diff
@@ -110,6 +110,9 @@ pre-aggregations.
 
 ### Batching
 
+Cube requires the Redshift user to have ownership of a schema in Redshift to support pre-aggregations.  
+By default this schema is named `prod_pre_aggregations` but it can be customized using the environment variable `CUBEJS_PRE_AGGREGATIONS_SCHEMA` ([reference][env-vars-pre-agg-schema]).
```

Comment:
```suggestion
By default this schema is named `prod_pre_aggregations` but it can be customized using the `CUBEJS_PRE_AGGREGATIONS_SCHEMA` environment variable.
```

Reply:
@tonykau1 Just FYI, there's no need to add links to env vars explicitly, they are auto-detected and auto-linked 😄